tests/test_mt2.py

```diff
@@ -1,98 +1,114 @@
 """Tests for `mt2` package."""
+
 import math
 import random
+import unittest
 
-import pytest
+import numpy
 
 from mt2 import mt2, mt2_arxiv
 
 
-def test_simple_example():
-    for mt2_impl in (mt2, mt2_arxiv):
-        computed_val = mt2_impl(100, 410, 20, 150, -210, -300, -200, 280, 100, 100)
-        assert computed_val == pytest.approx(412.628)
-
-
-def test_near_massless():
-    # This test is based on Fig 5 of https://arxiv.org/pdf/1411.4312.pdf
-    m_vis_a = 0
-    px_a = -42.017340486
-    py_a = -146.365340528
-
-    m_vis_b = 0.087252259
-    px_b = -9.625614206
-    py_b = 145.757295514
-
-    px_miss = -16.692279406
-    py_miss = -14.730240471
-
-    chi_a = 0
-    chi_b = 0
+class TestMt2(unittest.TestCase):
+    def test_simple_example(self):
+        for mt2_impl in (mt2, mt2_arxiv):
+            computed_val = mt2_impl(100, 410, 20, 150, -210, -300, -200, 280, 100, 100)
+            self.assertAlmostEqual(computed_val, 412.627668458219)
```

Comment:
Changes in these files are just indentation, and replacing some pytest-specific calls with unittest or numpy approximate comparisons as appropriate.